src/wizard/javascript/browser.md

```diff
@@ -5,28 +5,39 @@ support_level: production
 type: language
 ---
 
-The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
-
-```html
-<script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
-  crossorigin="anonymous"
-></script>
+Install our JavaScript browser SDK using either `yarn` or `npm`:
+
+```bash {tabTitle: ESM}
+# Using yarn
+yarn add @sentry/browser @sentry/tracing
+# Using npm
+npm install --save @sentry/browser @sentry/tracing
 ```
 
-You should `init` the Sentry Browser SDK as soon as possible during your page load:
+We also support alternate [installation methods](/platforms/javascript/install/).
+
+`init` the Sentry Browser SDK as soon as possible during your page load:
 
 ```javascript
-Sentry.init({ dsn: "___PUBLIC_DSN___" });
+import * as Sentry from "@sentry/browser";
+import { Integrations } from "@sentry/tracing";
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new Integrations.BrowserTracing(),
+  ],
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
+});
 ```
 
-One way to verify your setup is by intentionally causing an error that breaks your application.
-
-Calling an undefined function will throw an exception:
+This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:
 
 ```js
 myUndefinedFunction();
 ```
 
-You can verify the function caused an error by checking your browser console.
+To view and resolve this intentional error, log into sentry.io and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
```

Comment:
I feel like we're introducing concepts that people aren't ready to wrap their heads around just yet. What is an `error title`? I'm already logged in; why am I being instructed to open sentry.io and log in? 

Note that if they send this error, they will get an email that will bring them into their account, we don't need to give them instructions that they'll forget.

Reply:
I agree in this context. Will edit to:
"Sentry will send you an email alerting you to this intentional error; follow the instructions in the email to resolve the error."